mysql/const.go

```diff
@@ -44,6 +44,13 @@ const (
 	ServerPSOutParams              uint16 = 0x1000
 )
 
+// Server limitations
+const (
+	ServerTableNameLength    int = 64
```

Comment:
I thought it is related to server scope so added the prefix. I will remove it in next commit.